internal/credential/vault/repository_credential_library.go

```diff
@@ -10,6 +10,15 @@ import (
 	"github.com/hashicorp/boundary/internal/errors"
 	"github.com/hashicorp/boundary/internal/kms"
 	"github.com/hashicorp/boundary/internal/oplog"
+	"github.com/hashicorp/boundary/sdk/strutil"
+)
+
+const (
+	nameField = "Name"
+	descriptionField = "Description"
+	vaultPathField = "VaultPath"
+	httpMethodField = "HttpMethod"
+	httpRequestBodyField = "HttpRequestBody"
```

Comment:
This feels wrong to me. Creating string `const` values for fields means the fields are now declared in the protobuf, the struct the protobuf generator generates, and now as here. What value does this add?

Reply:
This helps avoid typo errors since some of these are used up to 5 times in this file.

Reply:
True. And avoiding typos is a good thing. It still feels wrong to me that we are hand rolling field names when we have structs with the names but I don't have a better solution right now.